backend/dal/Repositories/AccessRequestRepository.cs

```diff
@@ -29,10 +29,9 @@ public class AccessRequestRepository : BaseRepository<PimsAccessRequest>, IAcces
         /// </summary>
         /// <param name="dbContext"></param>
         /// <param name="accessRequest"></param>
-        /// <param name="service"></param>
         /// <param name="logger"></param>
         /// <returns></returns>
-        public AccessRequestRepository(PimsContext dbContext, System.Security.Claims.ClaimsPrincipal user, ClaimsPrincipal accessRequest, IPimsRepository service, ILogger<AccessRequestRepository> logger, IMapper mapper) : base(dbContext, user, service, logger, mapper) { }
+        public AccessRequestRepository(PimsContext dbContext, System.Security.Claims.ClaimsPrincipal user, ClaimsPrincipal accessRequest, ILogger<AccessRequestRepository> logger) : base(dbContext, user, logger) { }
```

Comment:
These where just general cleanup of the base classes givenrepositories dont need access to the general service or repository provider